components/base/properties.go

```diff
@@ -0,0 +1,36 @@
+// Package base contains an enum representing optional base features
+package base
+
+import pb "go.viam.com/api/component/base/v1"
+
+// Properties is an structure representing optional base features
+// of a base reporting its respective minimum turning radius and
+// width, in units of meters (M).
+type Properties struct {
+	TurningRadiusMeters float64
+	WidthMeters         float64
+}
+
+// ProtoFeaturesToMap takes a GetPropertiesResponse and returns
+// an equivalent Feature-to-boolean map.
```

Comment:
Comment doesn't match code: there is no feature-to-boolean map any more.

Reply:
Good catches, thanks!